### ISSUE TITLE
A11y_AzureDataStudio_Prerequisites_Keyboard:Prerequisites window screen is not accessible by the keyboard.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7676]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#SQLAzureDataStudio;#MAS2.1.1;#DesktopApp;#Win32;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on 
         select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next 
        button and click on script to Notebook ->Navigate till Prerequisites.
8.	Check the Prerequisites window screen is accessible by the keyboard or not.Here is is accessible by the mouse double click.

**Actual:**
Prerequisites window screen is not accessible by the keyboard.

**Expected:**
Keyboard focus should move to Prerequisites window screen and should  accessible by the keyboard.

**Same issue observed for the below screen:**
**1.Create Azure Kubernetes service cluster**
1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on
select button.
6. Select any block and click on next button and again click on next button.
7. Enter password and confirm password and click on next button and again click on next
button and click on script to Notebook.
8. Navigate till " Create Azure Kubernetes service cluster" .

**2.Check dependencies** 
1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on select button.
6. Select any block and click on next button and again click on next button.
7. Enter password and confirm password and click on next button and again click on next button and click on script to Notebook ->Navigate till Check dependencies.
8. Verify that all the elements of " Check dependencies " are accessible and MAS Compliant

**User Impact:**
Keyboard only user faced difficulty to access the control.

**Attachment for Reference:**
[7676_A11y_AzureDataStudio_Prerequisites_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3717638/7676_A11y_AzureDataStudio_Prerequisites_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
@corivera @chlafreniere  another notebook related issue

**Comment by chlafreniere** [comment]
This is a dupe of #7658 as it's due to the same underlying issue.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
